test/controllers/user_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class UserControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get user_index_url
+    assert_response :success
+  end
+
+  test "should get add" do
+    get user_add_url
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get user_edit_url
+    assert_response :success
+  end
+
+  test "should get delete" do
+    get user_delete_url
+    assert_response :success
+  end
+
```

Comment:
@IgorKrv please remove empty line and use rubocop

Reply:
fixed